client/extensions/woocommerce/woocommerce-services/views/label-settings/add-credit-card-modal.js

```diff
@@ -41,15 +39,17 @@ function AddCardDialog( {
 			isVisible={ isVisible }
 			onClose={ onClose }
 		>
-			<CreditCardFormWithStripe
-				createCardToken={ createCardAddToken }
-				recordFormSubmitEvent={ recordFormSubmitEvent }
-				saveStoredCard={ saveStoredCard }
-				successCallback={ onClose }
-				showUsedForExistingPurchasesInfo={ true }
-				heading={ translate( 'Add credit card' ) }
-				onCancel={ onClose }
-			/>
+			<StripeHookProvider configurationArgs={ { needs_intent: true } }>
+				<CreditCardForm
+					createCardToken={ createCardAddToken }
+					recordFormSubmitEvent={ recordFormSubmitEvent }
+					saveStoredCard={ saveStoredCard }
+					successCallback={ onClose }
+					showUsedForExistingPurchasesInfo={ true }
+					heading={ translate( 'Add credit card' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Add Credit Card' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/stripe-context-hook)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).